src/Axe.Windows.Telemetry/ExcludedException.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Telemetry
+{
+    /// <summary>
+    /// Class to wrap Exceptions that should be excluded from the Telemetry pipeline
```

Comment:
In one of these classes it might be nice to rationalize the inclusion of this exception at all. Why not just eat the exceptions and not throw this?

Reply:
I think this is clarified in the comments in iteration 6

---
In reply to: [277699439](https://github.com/Microsoft/accessibility-insights-windows/pull/331#discussion_r277699439) [](ancestors = 277699439)